pkg/security/resolvers/cgroup/model/types.go

```diff
@@ -61,3 +61,11 @@ func (ws WorkloadSelector) ToTags() []string {
 		"image_tag:" + ws.Tag,
 	}
 }
+
+// Copy returns a copy of the selector
+func (ws WorkloadSelector) Copy() *WorkloadSelector {
+	return &WorkloadSelector{
```

Comment:
`ws` is already a copy here since the receiver is not a pointer. Maybe you could return `&ws` directly ?
I'm not a fan of this kind of copy method where it's super easy to forget new fields

Reply:
:+1: